src/content/reference/cluster-upgrades/_index.md

```diff
@@ -103,7 +103,7 @@ Your Kubernetes workloads should continue to work as expected (given they meet a
 
 In particular this means:
 
-- Nodes will be drained, then stopped, then recreated.
+- Nodes are drained and recreated at the same time.
 - As a consequence of draining, Pods running on a node will be rescheduled to other nodes.
```

Comment:
Based on nodes now being drained and recreated at the same time, does this still apply?

Reply:
I think the original sentence was fine. 

A single machine will go through this steps and it's important to mention that a machine will be drained before recreation.

Reply:
oh ok! put it back to the original ✌️